server/map_tree_updater.go

```diff
@@ -67,7 +66,7 @@ func (t *mapTreeUpdater) update(ctx context.Context, tx storage.MapTreeTX, nodes
 		err = runTX(ctx, func(ctx context.Context, tx storage.MapTreeTX) error {
 			nodesCopy := make([]smt.Node, len(nodes))
 			copy(nodesCopy, nodes) // Protect from TX restarts.
-			acc := &txAccessor{hashes: hashes, preload: preload, tx: tx, rev: writeRev}
+			acc := &txAccessor{mtu: t, read: preload, tx: tx, rev: writeRev}
```

Comment:
give mtu a slightly more descriptive name?

Reply:
treeUpdater?

Reply:
Renamed to `updater`.